setup.py

```diff
@@ -291,10 +310,17 @@ def run(self):
     elif sys.platform == 'darwin':
         ext_modules.append(Extension('borg.platform_darwin', [platform_darwin_source]))
 
+
+def parse(root, describe_command=None):
+    return subprocess.check_output("git describe --tags --long").decode().strip()
```

Comment:
I think this is not working and should be removed. There is also a workaround on the installation documentation (echo to `_version.py`). `setuptools_scm` needs to be somehow disabled until it's fixed in upstream.